content/kubernetes/release-notes/k8s-6-4-2-3.md

```diff
@@ -0,0 +1,158 @@
+---
+Title: Redis Enterprise for Kubernetes release notes 6.4.2-3 (March 2023)
+linktitle: 6.4.2-3 (March 2023)
+description: 
+weight: 57
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [ 
+    /kubernetes/release-notes/k8s-6-4-2-3.md,
+    /kubernetes/release-notes/k8s-6-4-2-3/,   ]
+---
+## Overview
+
+The Redis Enterprise K8s 6.4.2-3 release supports the Redis Enterprise Software 6.4.2 and contains new features and feature improvements.
+
+The key features, bug fixes, and known limitations are described below.
+
+## Images
+
+* **Redis Enterprise**: `redislabs/redis:6.4.2-43`
+* **Operator**: `redislabs/operator:6.4.2-43`
+* **Services Rigger**: `redislabs/k8s-controller:6.4.2-43`
+
+### OpenShift images
+
+**OLM operator version**: `v6.4.2-3`
+
+* **Redis Enterprise**: `registry.connect.redhat.com/redislabs/redis-enterprise:6.4.2-43.rhel8-openshift`
+    (or `redislabs/redis-enterprise:6.4.2-43.rhel7-openshift` if upgrading from RHEL 7)
+* **Operator**: `registry.connect.redhat.com/redislabs/redis-enterprise-operator:6.4.2-3`
+* **Services Rigger**: `registry.connect.redhat.com/redislabs/services-manager:6.4.2-3`
+
+## Upgrade to 6.4.2-3
+
+This release uses a new file for the `ValidatingWebhookConfiguration`. To use the 6.4.2-3 release, delete the old file `redb-admission.yaml` and apply the new file `webhook.yaml`.
+
+## New features
+
+* Cluster level LDAP configuration support in REC (RED-83533)
+* Public preview support for declarative Active-Active configuration (RED-86470)(RED-86470)
+
+## Feature improvements
+
+* New labeling method for multi-namespace REDB (RED-83102)
+* Operator deployment uses built-in SCC in OpenShift (RED-90524)
+* Support for Redis Enterprise 6.4.2-30
+* Support added for K8s 1.26, EKS 1.24, AKS 1.25, Rancher 2.7/1.23-24
+* Improved log collector display and error handling
+
+## Bug fixes
+
+* Deployment manifests referenced non-existent redis-enterprise-admission ServiceAccount (RED-90469)
+* CVE-2022-1996 (RED-93026)
+* CVE-2022-28948 (RED-93027)
+* CVE-2021-44716 & CVE-2022-27664 (RED-93028)
+* CVE-2018-20225
+
+## API changes
+
+* Fields added to REC custom resource:
+  * `managedAPIs`
+  * `ldapClientCertificateSecretName`
+  * `LDAPAuthenticationQuery`
+  * `LDAPAuthorizationQuery`
+  * `LDAPQuery`
+  * `LDAPServer`
+  * `LDAPSpec`
+* New custom resource definitions for preview features:
+  * Redis Enterprise remote cluster (RERC)
+  * Redis Enterprise Active-Active database (REAADB)
+
+## Compatibility notes
+
+### Deprecated features
+
+Support for Gesher, the advanced admission control configuration, is deprecated.
```

Comment:
Another deprecation: the RedisEnterpriseCluster v1alpha1 API version:

"The **app.redislabs.com/v1alpha1** API version of the `RedisEnterpriseCluster` (REC) custom resource is deprecated and will be removed in a subsequent release. Customers should switch to use the **app.redislabs.com/v1** API version instead."